app/src/main/java/net/gsantner/markor/frontend/textview/HighlightingEditor.java

```diff
@@ -37,7 +38,7 @@
 import net.gsantner.opoc.wrapper.GsTextWatcherAdapter;
 
 @SuppressWarnings("UnusedReturnValue")
-public class HighlightingEditor extends AppCompatEditText {
+public class HighlightingEditor extends AppCompatMultiAutoCompleteTextView {
```

Comment:
`AppCompatMultiAutoCompleteTextView` Appears to have some performance enhancements. We don't actually use autocompletion or popups